index.js

```diff
@@ -268,21 +317,19 @@ async function runTask(options) {
   assert(options && typeof options.lambdaInput === 'object', 'options.lambdaInput object is required');
   assert(options.taskDirectory && typeof options.taskDirectory === 'string', 'options.taskDirectory string is required');
   assert(options.workDirectory && typeof options.workDirectory === 'string', 'options.workDirectory string is required');
+  assert(!options.layersDirectory || typeof options.layersDirectory === 'string', 'options.layersDirectory should be a string');
```

Comment:
There's refactoring to be done here, however that's outside the scope of the current ticket to some degree.    I opted for consistency here.